packages/react/src/_popover/Popover.tsx

```diff
@@ -33,6 +34,8 @@ export const Popover = forwardRefWithAs<'div', PopoverProps>(function Popover(
 				boxShadow: tokens.shadow.lg,
 				zIndex: tokens.zIndex.popover,
 				visibility,
+				minHeight: 'min-content',
+				overflow: 'hidden',
```

Comment:
We have `overflowY` above. Does this override that? If so, should it be removed?

Reply:
This is why things should be alphabetised… 📣

Reply:
Ooo yeah, good spot, let me look into these.

- [ ] alphabetise
- [ ] look into overflow use-cases